v23.1/cluster-setup-troubleshooting.md

```diff
@@ -409,7 +409,7 @@ CockroachDB's built-in disk stall detection works as follows:
 
 - Every time the storage engine writes to the main [`cockroach.log` file](logging.html#dev), the engine waits 30 seconds for the write to succeed (configurable with the `COCKROACH_LOG_MAX_SYNC_DURATION` environment variable). If the write to the log fails, the `cockroach` process is terminated and the following message is written to stderr / `cockroach.log`:
```

Comment:
How about something like this?

"If the write to the log fails, the `cockroach` process is terminated and the following message is written to stderr / `cockroach.log`, with details regarding the type of the write, size of the write, & duration that the write is ongoing for?"